src/components/DummyCheckin/VolunteerEventsTable.jsx

```diff
@@ -29,11 +29,14 @@ const RenderVolunteerRow = ({ volunteer, changeIsCheckedIn }) => {
     email,
     image_url,
     is_checked_in,
-    event_data_new_id,
+    number_in_party,
+    event_data_id,
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'event_data_id' is assigned a value but never used.